dev/PagerControl/InteractionTests/PagerControlTests.cs

```diff
@@ -35,6 +36,17 @@ public void TestCleanup()
             TestCleanupHelper.Cleanup();
         }
 
+
+        [TestMethod]
+        [TestProperty("TestSuite", "A")]
+        public void VerifyAxeScanPasses()
+        {
+            using (var setup = new TestSetupHelper("PagerControl-Axe"))
+            {
+                // AxeTestHelper.TestForAxeIssues();
```

Comment:
It would be better to disable this test using the annotation `[TestProperty("Ignore", "True")]` for clarity. Also a comment as to why it's disabled would also be helpful.